### PR TITLE
User quota

### PR DESCRIPTION
**Pull request by przybylski**
As a followup to to old PR https://github.com/owncloud/android-library/pull/44 new pull was made with fixes and comments from @davivel

### CODE REVIEW

File: src/com/owncloud/android/lib/resources/users/RemoteGetUserQuotaOperation.java [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    @Override
+    protected RemoteOperationResult run(OwnCloudClient client) {
+        RemoteOperationResult result = null;
+        int status;
+        GetMethod get = null;
+
+
+        //Get the user
+        try {
+            OwnCloudBasicCredentials credentials = (OwnCloudBasicCredentials) client.getCredentials();
```
Comment (malkomich) [inline src/com/owncloud/android/lib/resources/users/RemoteGetUserQuotaOperation.java]: It keeps only available for servers with authentication though Basic Auth

File: src/com/owncloud/android/lib/resources/users/RemoteGetUserQuotaOperation.java [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+import org.apache.commons.httpclient.NameValuePair;
+import org.apache.commons.httpclient.methods.GetMethod;
+import org.apache.http.HttpStatus;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+
+/**
+ * @author marcello
```
Comment (malkomich) [inline src/com/owncloud/android/lib/resources/users/RemoteGetUserQuotaOperation.java]: You can add yourself too if you want @przybylski  ;)

File: src/com/owncloud/android/lib/resources/users/RemoteGetUserQuotaOperation.java (hunk 1 above)
Comment (davivel) [inline src/com/owncloud/android/lib/resources/users/RemoteGetUserQuotaOperation.java]: Right now username may be accessed calling ```OwnCloudCredentials.getUsername()```, the casting is not needed anymore.

@przybylski , would you have the time to update it?

The rest seems OK.

**Comment by davivel** [comment]
@przybylski , if you don't mind, I will update by myself.

Thanks

**Comment by davivel** [comment]
Waiting for Travis.

@jesmrec , if you want to have a look to the test case...

**Comment by jesmrec** [comment]
@davivel the test case included is correct... no negatives in quotas. 

No problem for me to merge.

**Comment by davivel** [comment]
In there. Thanks to everybody.

**Pull request merged**